about.html

```diff
@@ -0,0 +1,122 @@
+      </div> -->
+    </nav>
+  </header>
+  <main class='about-container'>
+    <section class='about-headline'>
+      <h1 class='about-title'> CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <p class='about-detail'>
+        this system is designed to store, access and check projects done on the university with staff memebers and
+        students. enjoy the it to access projects done so far by accessing it here.
+      </p>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga fugit nulla minima consequatur nihil sint dolor officiis recusandae. Nostrum, asperiores voluptatum? Quod velit ea porro voluptatum voluptate excepturi consequatur assumenda.</p>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem praesentium nesciunt minima aliquam necessitatibus tempora qui voluptates, architecto, enim impedit doloribus! Nam tempora a sed perspiciatis corrupti sequi voluptatem odio?
+      </p>
+      <div class='address-info'>
+        <p>Please contact us per Email for any further questions about CC Global summit 2015!
+        </p>
+        <h4 class='email-address'>ccsummit2015@cckorea.org</h4>
+      </div>
+      <h4>
+        PCEHR Web Development Bootcamp Logo 
+      </h4>
+      <div class='logo-img'>
+        <img src="images/logo_1.png" alt="Logo">
+      </div>
+    </section>
+    <section class='past-events'>
+      <div class="event-title">
+        <h2>See the past Web BOOTCAMP</h2>
+        <span></span>
+      </div>
+      <p>Take a look at the past two Web Bootcamp which took place in Lagos and Abuja</p>
+      
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dolore ducimus molestiae corrupti, libero minus esse,
+          natus, eius assumenda atque neque minima deserunt iusto expedita dolor a ratione quis! Officiis, omnis.</p>
+        <div class='events'>
+          <div class="event">
+          <img src="images/conference1.jpg" alt="Event one">
+          <div class="event-description">
+            <h3>2020</h3>
+            <p>PCEHR Web Bootcamp in Lagos</p>
+          </div>
+        </div>
+        <div class="event">
+          <img src="images/conference2.jpg" alt="event two">
+          <div class="event-description">
+            <h3>2021</h3>
+            <p>Web Bootcamp</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div class='footer-section'>
+      <h3 class='footer-title'>Partner</h3>
+      <h4 class='indicator-footer'></h4>
+      <ul class='partners-list'>
+        <li class="partner"><a href="" class='mozila'>Mozila</a></li>
+        <li class="partner"><a href="" class='google'>Google</a> </li>
+        <li class="partner"><a href="" class='naver'>Naver</a></li>
+        <li class="partner"><a href="" class='daumk'>Daumkakao</a></li>
+        <li class="partner"><a href="" class='daumk'><i class="fa-brands fa-airbnb"></i> Airbnb</a></li>
+      </ul>
+    </div>
+  </footer>
```

Comment:
Great job 👏🏽
- [ ] Unfortunately, the design accomplished doesn't match the one shared in the [design project guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). see below the expected footer design for both the Home page and about us
**Home**
<img width="987" alt="Screenshot 2022-07-29 at 03 27 21" src="https://user-images.githubusercontent.com/31113416/181659271-03be967a-79e7-4b86-b916-b6edbf43affd.png">
**About**
<img width="874" alt="Screenshot 2022-07-29 at 03 27 11" src="https://user-images.githubusercontent.com/31113416/181659310-6976efe1-74c7-4436-8b63-40cb2c7b1844.png">